src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -234,9 +234,9 @@ private void Event(string SourceName, string EventName, IEnumerable<KeyValuePair
         /// It will eventually go away, but we should always reserve the ID for this.
         /// </summary>
         [Event(3, Keywords = Keywords.Events)]
-        private void EventJson(string SourceName, string EventName, string ArgmentsJson)
+        private void EventJson(string SourceName, string EventName, string ArgumentsJson)
```

Comment:
Technically I believe this ends up being a visible and potentially breaking change, as this name will show up as part of the manifest, be visible to consuming code via EventListener, etc.

Reply:
I agree, updating existing [Event]'ful method is generally a bad practice; manifest will change which can break multiple (not-too-obvious) scenarios. Thank you for catching this. :)